automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -1,14 +1,16 @@
 package org.broadinstitute.dsde.firecloud.api
 
 import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.automation.api.RestClient
+import org.broadinstitute.dsde.automation.config.AuthToken
 import org.broadinstitute.dsde.firecloud.api.WorkspaceAccessLevel.WorkspaceAccessLevel
-import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
-import org.broadinstitute.dsde.firecloud.util.Retry.retry
+import org.broadinstitute.dsde.firecloud.config.Config
```

Comment:
should this be `org.broadinstitute.dsde.automation.config.Config`?

Reply:
Here we do want to use the Firecloud version of Config because Orch tests need the FC-specific config